src/main/java/core/basesyntax/Main.java

```diff
@@ -1,11 +1,19 @@
 package core.basesyntax;
 
 import core.basesyntax.controler.ConsoleHandler;
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.factory.Factory;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
 
 public class Main {
-    public static void main(String[] args) {
-        ConsoleHandler consoleHandler = new ConsoleHandler();
+    public static void main(String[] args) throws InvocationTargetException,
+            NoSuchMethodException, InstantiationException, IllegalAccessException {
+        ConsoleHandler consoleHandler = (ConsoleHandler) Injector
+                .getInstance(ConsoleHandler.class);
         System.out.println("Enter value and risk");
-        consoleHandler.userHandler();
+        consoleHandler.betHandler();
+        BetDao betDao = Factory.getBetDao();
+        System.out.println("All bets: " + betDao.getAllBets());
```

Comment:
if u have both the two storages for Bet and User respectively, as well as two Dao implementations, you should use both and then output the result